### ISSUE TITLE
Harvest Totals only for Last 25 Harvested

### ISSUE THREAD

**Issue by ShawnLeary** [NewEconomyMovement/NemCommunityClient#379]
Is there a setting we can change to allow for "Total of all Harvested"?  Also possibly an "average XEM per harvest".

**Comment by gimer** [comment]
@BloodyRookie after all the inices that we've changed in db, do you think we could add query for that, that would be fast enough? (I think so)

**Comment by fyahfox** [comment]
What about a button "show total"?

**Comment by BloodyRookie** [comment]
The total and average harvested info should be only for the account displayed in the gui, right?
(An overall total/average for all blocks would be something to display in nembex maybe.)

**Comment by fyahfox** [comment]
Yes of course, the account which is currently selected in the GUI (NCC interface in the browser) should only be able to show the total harvested fees for that account, not all existing XEM accounts.
